### DISCUSSION TITLE
Problem when linking a branch named with UTF-8 characters.

### DISCUSSION THREAD

**Discussion by ryush00** [community/community#42824] [Actions]
### Select Topic Area

Bug

### Body

When `closing_references` action called with a branch named with UTF-8 characters, it fails with 422 error.
![image](https://user-images.githubusercontent.com/4997174/209916814-8b8d3e6e-562c-446d-8160-4b8781810edf.png)
(First request: `10004-codespace-정상-작동하도록-수정`, Second request: `10004-codespace-fix`)

### Example payload
```
...
branch_names[]: 10004-codespace-정상-작동하도록-수정
```

```json
{
  "error": "Invalid branch names"
}
```

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @ryush00 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
